mailbox/api/src/test/java/org/apache/james/mailbox/model/MailboxAssertingToolTest.java

```diff
@@ -157,7 +157,7 @@ void isEqualToShouldFailWithNotEqualId() {
         @Test
         void isEqualToShouldFailWithNotEqualUidValidity() {
             Mailbox mailbox1 = new Mailbox(MailboxPath.forUser(USER, "name"), UID_VALIDITY, MAILBOX_ID);
-            Mailbox mailbox2 = new Mailbox(MailboxPath.forUser(USER, "name"), UID_VALIDITY + 1, MAILBOX_ID);
+            Mailbox mailbox2 = new Mailbox(MailboxPath.forUser(USER, "name"), UidValidity.of(UID_VALIDITY.asLong() + 1), MAILBOX_ID);
```

Comment:
We should have a UidValidity factory to generate new ones.
The `+ 1` trick was just saying "I want a different one" not "I want the same value + 1".
So you can just write `UidValidity.create()` to have a more expressive test.